.github/workflows/ci_cd_release.yml

```
@@ -104,8 +122,10 @@ jobs:
     name: "Doc style"
     runs-on: ubuntu-latest
     needs: rolling-release
+    permissions:
+      contents: write
     steps:
-      - uses: ansys/actions/doc-style@main
+      - uses: ansys/actions/doc-style@feat/check-actions-security-action
```

```suggestion
      - uses: ansys/actions/doc-style@main
```
Needs to be reverted before merging